apps/condo/domains/resident/schema/RegisterResidentService.test.js

```diff
@@ -219,6 +209,55 @@ describe('manageResidentToPropertyAndOrganizationConnections worker task tests',
             expect(anotherResident.property.id).toEqual(olderProperty.id)
         })
     })
+
+    test('Must relink many residents after approve new properties', async () => {
```

Comment:
Do we really need this test? Locally, it takes about a minute to run, sometimes even longer, due to the creation of 101 residents

Reply:
Yea... I agree